src/audio/CMakeLists.txt

```diff
@@ -169,7 +169,8 @@ check_optimization(fma -mfma -ftree-vectorize -DOPS_FMA)
 check_optimization(hifi2ep -mhifi2ep "" -DOPS_HIFI2EP)
```

Comment:
Please split this into multiple patches. One patch per logical change.

Reply:
I've tried to split the patches. Please let me know if it's okay or not. Thanks.